### ISSUE TITLE
BUG: App crashes on folder add

### ISSUE THREAD

**Issue by JamesHeiney** [Ross-Patterson/Portals-Desktop-Organization#328]
### Check the Help Site and GitHub Issues/Discussions

- [X] I have checked and found nothing.

### Version

3.3.0.9

### Operating System

Windows 11

### What happened?

Folders were cleared when I rebooted this morning. Now the app crashes when I try to add the old folders back in. 
[portals-crashlog-20240715_0833.log](https://github.com/user-attachments/files/16234666/portals-crashlog-20240715_0833.log)

### Steps to Reproduce

Start app.
Try to add previous folder of shortcuts.

### What would you expect to happen?

I would expect it to work and not crash.

**Comment by JamesHeiney** [comment]
Also, have uninstalled and reinstalled. Installed as admin and tried running app as admin to no avail.

**Comment by Ross-Patterson** [comment]
Its saying that an absolute path is required, are you trying to create a Portal from a relative path somehow?

**Comment by JamesHeiney** [comment]
No, the folders are all on my c: drive. I hadn't changed anything. It was just a reboot that seemed to have broken it. Are there any settings in the background that might refer to a profile somewhere? We recently switched servers where personal profiles are kept where I work, but it was working after that switch up until Monday.
